core/model/src/driver.rs

```diff
@@ -115,12 +66,9 @@ impl PaymentConfirmation {
 // ************************** GET ACCOUNT BALANCE **************************
 
 #[derive(Clone, Debug, Serialize, Deserialize)]
-pub struct GetAccountBalance(String);
-
-impl From<String> for GetAccountBalance {
-    fn from(address: String) -> Self {
-        GetAccountBalance(address)
-    }
+pub struct GetAccountBalance {
+    pub platform: String,
```

Comment:
IMHO we do not need to send platform directly in message, it should be only specified in gsb router endpoint.

Reply:
This depends on whether we allow a single driver to support more than one platform. If no, then we can drop 'platform' parameter from most messages.